charts/ccsm-helm/templates/curator-configmap.yaml

```diff
@@ -0,0 +1,128 @@
+    # Also remember that all examples have 'disable_action' set to True.  If you
+    # want to use this action as a template, be sure to set this to False after
+    # copying it.
+    actions:
+      # delete zeebe- indicies
+      1:
+        action: delete_indices
+        description: "Clean up ES by deleting old Zeebe indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+          - filtertype: pattern
+            kind: prefix
+            value: zeebe-
+          - filtertype: age
+            source: name
+            direction: older
+            timestring: '%Y-%m-%d'
+            unit: days
+            unit_count: {{ .Values.retentionPolicy.zeebeIndexTTL }}
+            field:
+            stats_result:
+            epoch:
+            exclude: False
+      # delete operate- indicies
+      2:
+        action: delete_indices
+        description: "Clean up ES by deleting old Operate indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+          - filtertype: pattern
+            kind: prefix
+            value: operate-
+          - filtertype: age
+            source: name
+            direction: older
+            timestring: '%Y-%m-%d'
+            unit: days
+            unit_count: {{ .Values.retentionPolicy.operateIndexTTL }}
+            field:
+            stats_result:
+            epoch:
+            exclude: False
+      # delete tasklist- indicies
+      3:
+        action: delete_indices
+        description: "Clean up ES by deleting old Tasklist indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+          - filtertype: pattern
+            kind: prefix
+            value: tasklist-
+          - filtertype: age
+            source: name
+            direction: older
+            timestring: '%Y-%m-%d'
+            unit: days
+            unit_count: {{ .Values.retentionPolicy.tasklistIndexTTL }}
+            field:
+            stats_result:
+            epoch:
+            exclude: False
+      # or delete indicies which exceed the total size of 10 gig
+    {{- if .Values.retentionPolicy.zeebeIndexMaxSize }}
+      4:
+        action: delete_indices
+        description: "Clean up ES by deleting too big Zeebe indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+          - filtertype: pattern
+            kind: prefix
+            value: zeebe-
+          - filtertype: space
+            disk_space: {{ .Values.retentionPolicy.zeebeIndexMaxSize }}
+            source: name
+            timestring: '%Y-%m-%d'
```

Comment:
:clown_face: I thought filters could be merged, but I see from the docs that they are implicit AND :disappointed: